wgpu-native/src/command/render.rs

```diff
@@ -363,6 +369,12 @@ pub fn render_pass_set_vertex_buffers<B: GfxBackend>(
         .iter_mut()
         .zip(buffers.iter().zip(offsets))
     {
+        let buffer_usage = buffer_guard[id].usage;
```

Comment:
we get the buffer a few lines later, let's move the check there in order to avoid `buffer_guard` access here